compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/node/SortedSet.kt

```diff
@@ -16,7 +16,7 @@
 
 package androidx.compose.ui.node
 
-internal expect class TreeSet<E>(comparator: Comparator<in E>) {
+internal expect class SortedSet<E>(comparator: Comparator<in E>) {
```

Comment:
JFYI: it probably impacts android.
Could you please try to publish ui:ui for all targets to mavenLocal?
I suspect there can be an issue with commonMain metadata compilation.  Then we probably need to change the actulal in android sources too.

Reply:
Isn't `actual` for android picked up from `jvmMain` source set?

Reply:
Really? 
Android module ui:ui uses actual functions from jvmMain sourceSet?
Where is this rule configured?

Reply:
Based on upstream repo and search - yes. But it's configured not explicitly, so I cannot find specific place